src/components/Pagination.svelte

```diff
@@ -84,14 +93,48 @@
 
 <div class="flex flex-col items-center my-12">
   <div class="flex text-gray-700">
-    <div class="flex h-12 font-medium bg-gray-200">
-      {#each range(lastPage, 1) as page}
+    <div
```

Comment:
From [this comment](https://github.com/mozilla/glean-dictionary/pull/119#discussion_r508611891) I understand you didn't want the transition buttons in the original pagination implementation. I think having the buttons would be really helpful, especially for users with smaller screens, considering right now we don't have any pagination available at a small screen size.
<img width="898" alt="transition buttons" src="https://user-images.githubusercontent.com/28797553/102575842-0015a000-40a9-11eb-9d80-8e873134a9cc.png">

Reply:
Yup, sounds good-- I suggested taking them out of the initial implementation because it was already quite complex, but I think they make a nice addition here.